packages/client/src/components/panels/UploadPanel/SubComponents/LoadingButton.js

```diff
@@ -1,11 +1,83 @@
 import React, { useState, useEffect } from "react";
 import { inject, observer } from "mobx-react";
-import {
-  StyledCircle,
-  StyledLoadingButton,
-} from "@docspace/common/components/StyledLoadingButton";
+import styled, { css, keyframes } from "styled-components";
+import { ColorTheme, ThemeType } from "@docspace/components/ColorTheme";
+import { Base } from "@docspace/components/themes";
 
-import { ColorTheme, ThemeType } from "@docspace/common/components/ColorTheme";
+const rotate360 = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+const StyledLoadingButton = styled.div`
```

Comment:
Moved styled code to separated files

Reply:
Moved styles to one file StyledLoadingButton